proto/service/mixer.proto

```diff
@@ -611,6 +611,14 @@ service Mixer {
     };
   }
 
+  // Recognize non-place entities from a NL query.
+  rpc RecognizeEntities(RecognizePlacesRequest)
```

Comment:
Since these two apis share the same request and response, why not making them the same api?

Reply:
Following offline discussions, keeping the apis different, but added a new request and response